onnxruntime/test/optimizer/graph_transform_test_builder.cc

```diff
@@ -109,15 +109,15 @@ void TestGraphTransformer(const std::function<void(ModelTestBuilder& helper)>& b
               domain_to_version, {}, logger);
   Graph& graph = model.MainGraph();
   ModelTestBuilder helper(graph);
-  ASSERT_TRUE(build_test_case);
```

Comment:
Such ASSERT macros abort the **current** function, it has no effect on the callers of this function.  And the old version this function returns 'void', so its callers has no information on whether it was successful or not.